prediction_market_agent/agents/autogen_general_agent/deploy.py

```diff
@@ -0,0 +1,39 @@
+from loguru import logger
+from prediction_market_agent_tooling.deploy.agent import DeployableAgent
+from prediction_market_agent_tooling.markets.markets import MarketType
+
+from prediction_market_agent.agents.autogen_general_agent.social_agent import (
+    build_social_media_text,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.abstract_handler import (
+    AbstractSocialMediaHandler,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.farcaster_handler import (
+    FarcasterHandler,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.twitter_handler import (
+    TwitterHandler,
+)
+
+
+class DeployableFarcasterAgent(DeployableAgent):
```

Comment:
Since this agent posts to twitter as well as farcaster, should we give it a more generic name? e.g. `DeployableSocialMediaAgent`?

Reply:
accepted